frontend/src/component/executiveDashboard/componentsChart/MetricsSummaryChart/MetricsSummaryChart.tsx

```diff
@@ -1,36 +1,118 @@
 import { useMemo, type VFC } from 'react';
 import 'chartjs-adapter-date-fns';
+
 import type { ExecutiveSummarySchema } from 'openapi';
-import { LineChart, NotEnoughData } from '../../components/LineChart/LineChart';
+import {
+    fillGradientPrimary,
+    LineChart,
+    NotEnoughData,
+} from '../../components/LineChart/LineChart';
 import { MetricsSummaryTooltip } from './MetricsChartTooltip/MetricsChartTooltip';
 import { useMetricsSummary } from '../../hooks/useMetricsSummary';
 import { usePlaceholderData } from 'component/executiveDashboard/hooks/usePlaceholderData';
 import type { GroupedDataByProject } from '../../hooks/useGroupedProjectTrends';
+import { useTheme } from '@mui/material';
 
 interface IMetricsSummaryChartProps {
     metricsSummaryTrends: GroupedDataByProject<
         ExecutiveSummarySchema['metricsSummaryTrends']
     >;
+    isAggregate?: boolean;
+}
+
+function aggregateDataPerDate(
+    items: ExecutiveSummarySchema['metricsSummaryTrends'],
+) {
+    return items.reduce(
+        (acc, item) => {
+            if (!acc[item.date]) {
+                acc[item.date] = {
+                    totalApps: 0,
+                    totalEnvironments: 0,
+                    totalFlags: 0,
+                    totalNo: 0,
+                    totalRequests: 0,
+                    totalYes: 0,
+                };
+            }
+
+            acc[item.date].totalApps += item.totalApps;
+            acc[item.date].totalEnvironments += item.totalEnvironments;
```

Comment:
Could we end up with duplicates here? What if multiple items have the same app or environment? Or will that not happen?